tests/test_mypy.py

```diff
@@ -37,7 +37,7 @@
 @pytest.mark.skipif(not typing_extensions, reason='typing_extensions not installed')
 @pytest.mark.parametrize('filename,expected_result', expected_successes + expected_fails)
 def test_mypy_results(filename, expected_result):
-    actual_result = api.run([filename, '--config-file', 'tests/mypy/mypy-default.ini'])
+    actual_result = api.run([filename, '--config-file', 'tests/mypy/mypy-default.ini', '--no-error-summary'])
```

Comment:
New mypy feature -- prints out number of errors; disabled here so I don't need to update the tests.